project/deps.sc

```diff
@@ -180,16 +180,15 @@ object Deps {
   def scalaJsLogging   = ivy"org.scala-js:scalajs-logging_2.13:1.1.1"
   // Force using of 2.13 - is there a better way?
   def scalaJsTestAdapter = ivy"org.scala-js:scalajs-sbt-test-adapter_2.13:${Scala.scalaJs}"
-  // Force using of 2.13 - is there a better way?
-  def scalametaTrees = ivy"org.scalameta:trees_2.13:${Versions.scalaMeta}"
```

Comment:
So we never really needed trees actually? This actually seems like the wrong dependency was used

Reply:
It seems we only needed it for the SemanticDB stuff, yeah.